### ISSUE TITLE
Documentation issue: wrong file name in TLS instructions

### ISSUE THREAD

**Issue by peterbecker** [apache/incubator-pulsar#2530]
#### Expected behavior

Following the instructions for generating a server certificate at https://pulsar.incubator.apache.org/docs/latest/security/tls-transport/#Servercertificate-av2fql should provide me with a server certificate.

#### Actual behavior

The step labelled "… and sign it with the certificate authority." fails with:

```
proxy.csr.pem: No such file or directory
```

#### Steps to reproduce

Follow the instructions on the page.

#### System configuration

**Pulsar version**: n/a

#### Solution

The step labelled "Generate the certificate request…" needs to be adjusted to use the "broker.csr.pem" file name as last argument, not "broker.cert.pem" - it's generating the CSR after all. It should read:

```bash
openssl req -config openssl.cnf \
      -key broker.key.pem -new -sha256 -out broker.csr.pem
````

**Comment by sijie** [comment]
@peterbecker I think you are using an old website link, can you try using the instructions here: https://pulsar.incubator.apache.org/docs/en/security-tls-authentication/ ?

**Comment by peterbecker** [comment]
The page you link is the TLS auth page. Mine is the encryption one. You can follow the link on the page you linked, which takes you to https://pulsar.incubator.apache.org/docs/en/security-tls-transport/ which still has the issue.

**Comment by ivankelly** [comment]
@peterbecker you're right. Will update.

**Comment by sijie** [comment]
This is fixed by #2531
